### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by ad9311**
## Greetings!

In this PR, I present the following changes:

- Add a folder called components.
- Inside components, there is a file called **Calculator.js.**
- Calculator.js contains a **class component** of the same name that renders the calcularor.
- This component is called on **App.js.**
- Add stylesheets for Calculator.js and App.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
+import './app.css';
 
 function App() {
```
Comment (oxenprogrammer) [inline src/App.js:4]: **Changes Required**
Good effort with the function component, however;
- [ ] The `App Component` should be a `Class Component` as specified in the project requirement. Let us refactor it into a class component.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @ad9311 , 

Good job so far!, I must applaud your efforts.
**To Highlight**
- [x] Good UI.
- [x] Correct Gitflow.

   
I can see you're really putting effort to become a world-class software developer :+1:
There are some issues that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

- [ ] I will advice that you might consider removing the auto generated section from your README file and only stick to your custom edited. Those changelog has nothing to do with your project. Thank you

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by ad9311** [comment]
**Changes:**
- As mentioned by the previous reviewer I have made App.js as a class component.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**